src/poprox_recommender/training/train.py

```diff
@@ -144,11 +144,20 @@ def save_model(model):
     parser.add_argument("--num_words_title", type=float, default=30)
 
     args = parser.parse_args()
+
+    # set a seperate output directory for subset
+    if args.subset is not None:
+        args.output_dir = root / f"models/nrms-mind-subset-{args.subset}"
+    else:
+        args.output_dir = MODEL_DIR
+
+    os.makedirs(args.output_dir, exist_ok=True)
+
     lc = LoggingConfig()
     if args.verbose:
         lc.set_verbose(True)
     lc.apply()
 
     torch.cuda.empty_cache()
 
-    train(device, args.load_checkpoint, args)
+    train(device, args.load_checkpoint, args, args.output_dir)
```

Comment:
Since we're passing `args`, why not just have `train` look at it instead of passing `args.output_dir` separately?